Data/Preprocessed/README.md

```diff
@@ -0,0 +1,23 @@
+# Preprocessed README
+
+Note: all information for mappings from measure columns and files to measure names can be found in Data/Metadata.xlsx. See Data README.md for more information. All data in the Preprocessed folder is organized by data source. See Documentation folder for information on each measure.
+
+Folders in "Preprocessed" are as follows:
+- 2021_release: contains preprocessed measure files used in 2021_release, if they have been updated. Not updated files remain in main folder.
```

Comment:
Same comment:

2021_release: contain older versions of preprocessed measure files used in prior 2021 MWI release. Files without data updates remain in main folder.